llvm-external-projects/iree-dialects/BUILD

```diff
@@ -294,7 +294,7 @@ cc_library(
         "@llvm-project//mlir:DialectUtils",
         "@llvm-project//mlir:FuncDialect",
         "@llvm-project//mlir:IR",
-        "@llvm-project//mlir:InferTypeOpIntefaces",
+        "@llvm-project//mlir:InferTypeOpInteface",
```

Comment:
Inteface -> Interface? I am curious if the typo must be maintained to work properly.

Reply:
Oh thats why CI is failing. Thanks!